spanner/google/cloud/spanner/database.py

```diff
@@ -389,8 +389,7 @@ def batch(self):
         """
         return BatchCheckout(self)
 
-    def snapshot(self, read_timestamp=None, min_read_timestamp=None,
-                 max_staleness=None, exact_staleness=None):
+    def snapshot(self, **kw):
```

Comment:
@tseaver It's fine to leave as-is.

FWIW "our own code uses it 430 times" isn't much of an argument if you are the author of all 430 instances.